app/Repositories/LocationRepository.php

```diff
@@ -92,9 +95,9 @@ public function getDatesOfFilter($filter)
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Variables.ValidVariableName.NotCamelCaps</br>**Severity**: error</br>**File**: app/Repositories/LocationRepository.php</br>Variable "array_dates" is not in valid camel caps format